meerk40t/core/elements.py

```diff
@@ -3658,7 +3658,7 @@ def element_fill(command, channel, _, color, data=None, filter=None, **kwargs):
         @context.console_command(
             "outline",
             help=_("outline the current selected elements"),
-            input_type=(
+            input_type=(            
```

Comment:
A very minor point, but you have added trailing spaces here - you might want to set your editor to remove trailing spaces on save.